src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -530,7 +530,7 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    *
    * @todo Use a custom exception type rather than std::domain_error.
    */
-  [[nodiscard]] EntityReference createEntityReference(Str entityReferenceString) const;
+  [[nodiscard]] EntityReference createEntityReference(Str entityReferenceString);
```

Comment:
[question] Presumably the `const` was removed here just because internally it needs to use `isEntityReference`?  Just checking since it's against the ACs.

Reply:
I think this is pretty obviously the reason, will resolve (Tom is away, I'm taking over).